public/app/features/plugins/admin/api.ts

```diff
@@ -63,22 +62,11 @@ async function uninstallPlugin(id: string) {
   return await getBackendSrv().post(`${API_ROOT}/${id}/uninstall`);
 }
 
-async function updatePlugin(pluginId: string, data: Partial<PluginMeta>) {
```

Comment:
This is redundant and  left over from when the Catalog was a Grafana app plugin and had a settings page.